contracts/DNSClaimChecker.sol

```diff
@@ -53,26 +53,32 @@ library DNSClaimChecker {
         return 0;
     }
 
-    function parseRR(bytes memory rdata, uint idx) internal pure returns (address) {
+    function parseRR(bytes memory rdata, uint idx, address defaultAddr) internal pure returns (address) {
```

Comment:
For consistency, why not make this return an (address, found) tuple too, instead of taking a default?

Reply:
I guess we can do that yeah.

Reply:
I will change the entire code to reflect that, and then remove the actual defaulAddr code. Changing the interface of `getLabels` too. Then the default handling goes there where it would be required.